From c56b53d62116b4672962124afab02f3beada4244 Mon Sep 17 00:00:00 2001
From: Tomi Valkeinen <tomi.valkeinen@ideasonboard.com>
Date: Fri, 29 Sep 2023 12:57:23 +0300
Subject: [PATCH] media: rp1: csi2: Set values for enum csi2_mode

Set hardcoded values for enum csi2_mode, as the values will be
programmed to HW registers.

Signed-off-by: Tomi Valkeinen <tomi.valkeinen@ideasonboard.com>
---
 drivers/media/platform/raspberrypi/rp1_cfe/csi2.h | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

--- a/drivers/media/platform/raspberrypi/rp1_cfe/csi2.h
+++ b/drivers/media/platform/raspberrypi/rp1_cfe/csi2.h
@@ -20,10 +20,10 @@
 #define DISCARDS_TABLE_NUM_VCS 4
 
 enum csi2_mode {
-	CSI2_MODE_NORMAL,
-	CSI2_MODE_REMAP,
-	CSI2_MODE_COMPRESSED,
-	CSI2_MODE_FE_STREAMING
+	CSI2_MODE_NORMAL = 0,
+	CSI2_MODE_REMAP = 1,
+	CSI2_MODE_COMPRESSED = 2,
+	CSI2_MODE_FE_STREAMING = 3,
 };
 
 enum csi2_compression_mode {
